components/03-modules/range/range.config.js

```diff
@@ -27,6 +28,8 @@ module.exports = {
       },
       'field': {
```

Comment:
delete this " ' " it's not needed in config

Reply:
sure, removed.